proto/internal/temporal/server/api/persistenceblobs/v1/message.proto

```diff
@@ -41,6 +41,23 @@ import "temporal/server/api/enums/v1/workflow.proto";
 import "temporal/server/api/enums/v1/task.proto";
 import "temporal/server/api/replication/v1/message.proto";
 
+
+// ExecutionStats is not persisted and is used internally and as part of mutableState
+message ExecutionStats {
+    int64 history_size = 1;
+}
+
+// ReplicationState represents mutable state information for global namespaces.
+// This information is used by replication protocol when applying events from remote clusters
+// (not persisted)
+message ReplicationState {
+    int64 current_version = 1;
+    int64 start_version = 2;
+    int64 last_write_version = 3;
+    int64 last_write_event_id = 4;
+    map<string, temporal.server.api.replication.v1.ReplicationInfo> last_replication_info = 5;
```

Comment:
I think they should be moved to another proto packages (i.e. `workflow` and `replication`) but I will do it myself later.